scalableminds/webknossos
# Fix screenshot tests for real

Fixed some errors in the screenshot tests. The URL was not properly joined sometimes, some globals were missing in the node context and I've switched the tests to be run serially as chrome was crashing due to RAM shortage otherwise (RAM usage was increased by the fullPage screenshots, which use a lot of RAM according to an issue in the puppeteer repo). The tests don't run significantly longer though, still ~6.5 minutes, so that's not a problem :)

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- successful CI run should be enough (I configured the screenshot tests to be run for every build)

### Issues:
- fixes screenshot tests

------
- [x] **Remove nightly job again before merging**
